docs/rules/path.md

```diff
@@ -0,0 +1,54 @@
+# import/path
+
+🔧 This rule is automatically fixable by the [`--fix` CLI option](https://eslint.org/docs/latest/user-guide/command-line-interface#--fix).
+
+<!-- end auto-generated rule header -->
+
+Enforce paths of `require()` / `import` statements to be "absolute" or "relative".
+
+## Rule Details
+This rule enforces module paths in `require()` / `import` statements to be "relative"
+or "absolute" (relative to the project root).
```

Comment:
The word "absolute" means "relative to `/`", so I very much think this is the wrong term - absolute paths are a horrifically bad practice and should never be used.

How can a path be relative to the project root when there does not exist a generic mechanism for descending from the project root?

Reply:
Changed "absolute" to "relativeToRoot" (root means project root).
Changed "relative" to "relativeToCurrent".